ckcp/openshift_dev_setup.sh

```diff
@@ -305,9 +305,13 @@ patches:
   # Workaround to prevent the creation of a new workspace until KCP is ready.
   # This fixes `error: creating a workspace under a Universal type workspace is not supported`.
   ws_name=$(echo "$kcp_org" | cut -d: -f2)
-  while ! KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workspace create "$ws_name" --type root:organization --ignore-existing >/dev/null; do
-    sleep 5
-  done
+  local sec=0
+  if  [ "$sec" -lt 100 ]; then
+    while ! KUBECONFIG="$KUBECONFIG_KCP" kubectl kcp workspace create "$ws_name" --type root:organization --ignore-existing >/dev/null; do
+      sleep 5
```

Comment:
I guess you meant to have the if inside of the loop with a break when it is true

Reply:
Fixed.